math/src/field/f64/mod.rs

```diff
@@ -99,51 +99,33 @@ impl FieldElement for BaseElement {
     #[inline]
     #[allow(clippy::many_single_char_names)]
     fn inv(self) -> Self {
-        let x = self.as_int();
+        // compute base^(M - 2) using 72 multiplications
+        // M - 2 = 0b1111111111111111111111111111111011111111111111111111111111111111
 
```

Comment:
// t_i represents base^{2^i}

Reply:
kind of: up until `t31` that's how it works. beyond that, the number is just for the number of bits of the exponent included in the temp value.

Reply:
kind of: up until `t31` that's how it works. beyond that, the number is just for the number of bits of the exponent included in the temp value.